xml/hardware_nvdimm.xml

```diff
@@ -38,33 +39,34 @@
   <para>
    Like SSDs, NVDIMMs provide non-volatile storage: their contents are retained
    when the system is powered off or restarted. Also like SSDs, sector-level
-   access is also possible if that is more suitable for a particular
-   application.
+   access is possible if that is more suitable for a particular application.
   </para>
 
   <para>
    This new storage subsystem is defined in version 6 of the ACPI standard.
-   However, some types of NVDIMMs already existed before the standard was
-   defined. These can be used in the same way.
-  </para>
-  
-  <para>
-   Different models use different forms of electronic storage medium, such as
-   Intel 3D XPoint, or a combination of NAND flash and DRAM, and in future,
-   memristor storage or other forthcoming types of non-volatile memory.
+   However, <filename>libnvdimm</filename> supports pre-standard NVDIMMs and
+   they can be used in the same way.
   </para>
 
   <para>
-   This means that different vendors' NVDIMMs offer different performance and
-   durability characteristics. Typically, they are slower than DRAM, but
-   considerably faster than SSDs based on NAND-flash technology.
+   Different models use different forms of electronic storage medium, such as
+   Intel 3D XPoint, or a combination of NAND flash and DRAM. New forms of
+   non-volatile RAM are also in development. This means that different vendors'
+   NVDIMMs offer different performance and durability characteristics.
   </para>
-
+  
   <para>
    Because the storage technologies involved are in an early stage of
    development, different vendors' hardware may impose different limitations.
-   However, generally, writing data to NVDIMMs is slower than reading from
-   them, and the number of write cycles is generally limited.
+   Thus, the following statements are generalizations.
+  </para>
+  
+  <para>
+   Persistent memory is up to ten times slower than DRAM, but around a thousand
+   times faster than flash storage. Like SSDs, writing is slower than reading,
+   and it can only handle a finite number of rewrite cycles. However, most forms
```

Comment:
The sentence 'Like SSDs, ...'  is a bit ambiguous. Does the limitation ' can only handle a finite number of review cycles' apply to both persistent memory *and* to SSDs? If yes, maybe rephrase the sentence to:
' Like (with?) SSDs, writing to persistent memory is slower than reading from it it, and both SSDs and persistent memory can only handle a finite number of rewrite cycles. ' 
(not sure about adding 'with' - up to you to decide as native speaker) :)

Reply:
This one is more difficult. I have used the phrase "Like SSDs" 3 times, in 2 places. I did not realize that it could be ambiguous. I will rewrite both sections that compare the 2 forms of storage and attempt to remove any ambiguity.

Reply:
This one is more difficult. I have used the phrase "Like SSDs" 3 times, in 2 places. I did not realize that it could be ambiguous. I will rewrite both sections that compare the 2 forms of storage and attempt to remove any ambiguity.